release/ray_release/tests/test_bisect.py

```diff
@@ -15,6 +15,7 @@ def test_bisect():
         "c1": {
             "c0": "passed",
             "c1": "hard_failed",
+            "c2": "hard_failed",
```

Comment:
could you add tests for:

- just one single commit, or just two commits
- all commits failed
- all commits passed (or do we assume that the first one is always passing and last one is always failing?)

Reply:
good point; i think we can assume the boundary is passing and failing as well (we sanity checked this), otherwise bisect will be return meaningless results